app/src/App.js

```diff
@@ -37,7 +37,7 @@ export default function App() {
     <Switch>
       <Route path="/" render={ routeProps => {
         const params = parseParams(routeProps.location.search);
-        if (params.k && params.v) {
+        if ((params.k && params.v) || params.id) {
           return (
```

Comment:
What if params are null or undefined?
Can we add a null check for params as well, something like below
let paramObj = params ? params : {} 
if ((paramObj.k && paramObj.v) || paramObj.id) {

Reply:
ParseParams will not return null or undefined. It returns Array.reduce seeded with an empty Object.